docs/development-guide.md

```diff
@@ -24,6 +24,8 @@ this relative directory layout.
 - docker
 - make
 - Python 3 (only for needed for deploying)
+- npm
```

Comment:
I don't think `npm` is really _required_ because the only uses of it can be done inside of docker containers that have the correct version of node installed. And I don't think I'd say `npm` in any case I'd probably say `node`. But having `node 10` or probably shortly `node 12` installed is certainly a nice to have (https://nodejs.org/en/download/).

Reply:
At one point in the  process, an "npm install" needs to be run locally in the signalmaster directory, rather than inside a container. Changing it to node makes sense though.

Reply:
Huh, I forgot it still said that. It's no longer true (I think I updated the doc in edu-docker but didn't make the changes here).

Reply:
Ok. Sounds like I should I also remove the npm install  instructions for signalmaster.